qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -93,6 +132,270 @@ def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] =
         self._hash_to_counter_map = {}
         self._parameter_counter = 0
 
+        self._library = None
+        if library is not None:
+            self._library = library
+
+            # Add the basis gates
+            for gate in library.basis_gates:
+                self.add_schedule(library[gate], num_qubits=library.num_qubits(gate))
+
+            # Add the default values
+            if add_parameter_defaults:
+                for param_conf in library.default_values():
+                    schedule_name = param_conf[-1]
+                    if schedule_name in library.basis_gates:
+                        self.add_parameter_value(*param_conf)
+
+        # Instruction schedule map variables and support variables.
+        self._inst_map = InstructionScheduleMap()
+        self._operated_qubits = defaultdict(list)
+
+        # Use the same naming convention as in backend.defaults()
+        self.qubit_freq = Parameter(self.__qubit_freq_parameter__)
+        self.meas_freq = Parameter(self.__readout_freq_parameter__)
+        self._register_parameter(self.qubit_freq, ())
+        self._register_parameter(self.meas_freq, ())
+
+        # Backends like Armonk do not have a coupling map.
+        num_qubits = max(max(coupling_map)) + 1 if coupling_map is not None else 1
+
+        self._qubits = list(range(num_qubits))
+        self._coupling_map = coupling_map
+
+        self.update_inst_map_on_add = True
+
+        # Push the schedules to the instruction schedule map.
+        self.update_inst_map()
+
+    @classmethod
+    def from_backend(
+        cls,
+        backend: Backend,
+        library: Optional[BasisGateLibrary] = None,
+        add_parameter_defaults: bool = True,
+    ) -> "Calibrations":
+        """Create an instance of Calibrations from a backend.
+
+        Args:
+            backend: A backend instance from which to extract the qubit and readout frequencies
+                (which will be added as first guesses for the corresponding parameters) as well
+                as the coupling map. If the backend is not given then the calibrations will be
+                initialized with the following key-word arguments: control_config, coupling_map,
+                num_qubits.
```

Comment:
I think there is some carried over text here:

```suggestion
                as the coupling map.
```

Reply:
Yup. Thanks!